.travis.yml

```diff
@@ -10,4 +10,5 @@ before_script:
   - docker images --no-trunc
 script:
   - npm test
+  - git clone https://github.com/Khan/KaTeX-test-fonts test/screenshotter/unicode-fonts
```

Comment:
Do you want to clone a specific commit of that repo, so that you can update the fonts without breaking existing screenshots, and then can update the commit id and the screenshots in one go?